components/about/Tweet.vue

```diff
@@ -43,20 +43,39 @@
 export default {
   props: {
     tweet: {
-      required: true,
       type: Object,
+      required: true
     },
+    turquoise: {
+      type: Boolean,
+      required: false
+    }
   },
+  methods: {
+    avatarColorClasses(turquoise) {
```

Comment:
it might be easier to make conditional components/elements instead of manipulating so many classes. Would probably be easier for the next contributor to read

Reply:
I think you're right here; I think I was approaching it from a 'what if we have multiple conditions' standpoint, but we only have the one, so we can just ternary here.

Reply:
Ok, I take that back. I think we need to do it this way to keep the other colors on there.

The above comment was made on the assumption that static and interpolated values would be combined, like with css classes in Vue. That would look something like this:
`color="lighten-2 avatar-border" :color="turquoise ? 'white' : 'cyan'"`

However, that doesn't work, likely because of the way that Vuetify implements the avatar component.

The other way of course could be swapping out whole elements, but I think that's way more confusing as it's a bunch of duplication for no apparent purpose.